Demo/Demo/SwiftUIComponents/CardPaymentViews/CardOrderApproveView.swift

```diff
@@ -6,6 +6,7 @@ struct CardOrderApproveView: View {
 
     let cardSections: [CardSection] = [
         CardSection(title: "Successful Authentication Visa", numbers: ["4868 7194 6070 7704"]),
+        CardSection(title: "Vault with Purchase(no 3DS)", numbers: ["4000 0000 0000 0002"]),
```

Comment:
```suggestion
        CardSection(title: "Vault with Purchase (no 3DS)", numbers: ["4000 0000 0000 0002"]),
```
🧶 : take or leave.

Reply:
➕ on adding the space!

Reply:
Addressed in e2f1c5967aa90e94f758493b70ef36e2732e78b3